geniza/corpus/templates/corpus/snippets/document_image_rights.html

```diff
@@ -10,23 +10,35 @@
                     {% with fragment.url|yesno:"a,span" as tag %}
                         <{{ tag }} class="shelfmark"{% if tag == "a" %} href="{{ fragment.url }}" target="_blank"{% endif %}>{{ fragment.shelfmark }}</{{ tag }}>:
                     {% endwith %}
-                    {{ fragment.attribution|default_if_none:"" }}
+                    {% if fragment.attribution %}
+                        {{ fragment.attribution }}
+                    {% elif fragment.provenance %}
+                        {{ fragment.provenance }}
+                    {% endif %}
                     {% if fragment.manifest.license %}
                         {% include "corpus/snippets/fragment_license_statement.html" %}
-                    {% elif not fragment.attribution %}
+                    {% elif not fragment.attribution and not fragment.provenance %}
                         {% translate "No attribution or license noted." %}
                     {% endif %}
                 </li>
             {% endif %}
         {% endfor %}
     </ul>
     {% regroup document.fragments.all by manifest.logo as logos_list %}
+    {% regroup document.fragments.all by provenance as provenance_list %}
     <ul class="logos">
         {% for logo in logos_list %}
             <li>
                 <img src="{{ logo.grouper }}" alt="" role="presentation" height="64" loading="lazy" />
             </li>
         {% endfor %}
+        {% for provenance in provenance_list %}
+            {% if "JTS" in provenance.grouper %}
+                <li>
+                    <img src="{% static 'img/logos/all/all/jts-logo.png' %}" alt="" role="presentation" height="64" loading="lazy" />
```

Comment:
Since we're embedding this directly, let's actually provide alt text and link to the JTS website. (Similar to what we would expect to get in `provider` information in v3 of iiif presentation api)

https://www.jtsa.edu/

Reply:
Since we need to switch colors for light and dark mode with the new logo, I used `svg`/`use` instead. I still provided the link and accessibility labels, hopefully that is sufficient.

Reply:
Didn't think about that! Good idea for future cases like this.